docs/one-click-integrations/jira-integration.md

```diff
@@ -1,29 +1,96 @@
 # Jira integration
 
-Pulse integrates with Jira Cloud to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+Pulse integrates with Jira Cloud to receive data about:
+
+-   **Issues** - necessary to calculate [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+
+-   **Incidents** - necessary to calculate the following Accelerate metrics:
+
+    -   [Time to recover](../metrics/accelerate.md#time-to-recover)
+
+    -   [Change failure rate](../metrics/accelerate.md#change-failure-rate)
 
 ## Setting up the Jira integration
 
+!!! note
+    -   The Jira integration must be set up by a **Jira administrator**, as Pulse will only have access to Jira resources that the user setting up the integration has access to.
+    -   You can only integrate one Pulse organization with each Jira instance.
+    -   If you want to report incidents using Jira, check the [details of the incident detection strategy](#incident-detection-strategy) before you proceed with the integration setup.
+    -   [See below](#jira-permissions) the permissions that Pulse requires from your Jira instance.
+
 To set up the Jira integration:
 
 1.  On Pulse, [expand **Integrations** and select **Jira**](https://app.pulse.codacy.com/integrations/jira){: target="_blank"}.
 
-1.  Click **Install Jira** and follow the instructions on the Jira UI to install the app.
+1.  Click **Install Jira App** and follow the instructions on the Jira UI to install the app. You must be a Jira administrator to proceed with this step.
 
-    !!! important
-        This step should be performed by a Jira administrator, since Pulse will only have access to Jira resources that the user setting up the integration has access to.
+    ![Installing the Pulse Jira app](images/jira-installing.png)
 
-        You can only integrate one Pulse organization with each Jira instance.
+1.  Confirm that Pulse successfully connected the integration to Jira. If there is an error please [contact support](mailto:pulsesupport@codacy.com).
 
-    ![Installing the Pulse Jira app](images/jira-installing.png)
+    ![Pulse Jira App installed successfully](images/jira-installed-ok.png)
+
+1.  Choose the strategy to detect incidents that best fits your workflows. See the [section below](#incident-detection-strategy) for a detailed description of each option.
+
+    ![Configuring the incident detection strategy](images/jira-incident-strategy.png)
+
+1.  Click **Complete setup**.
+
+Your Jira integration is now complete. Pulse will start loading your data for the last 90 days.
+
+<!--TODO Add screenshot for setup ok-->
+
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The Pulse Jira integration can detect **incidents** automatically using the [issues assigned with the label **Incident**](#jira-incident-label).
```

Comment:
style: Since `Incident` is the exact label string that the users must use, I suggest the inline code formatting instead (like we already have below).

```suggestion
The Pulse Jira integration can detect **incidents** automatically using the [issues assigned with the label `Incident`](#jira-incident-label).
```